DoorsOpen/Views/Home/Index.cshtml

```diff
@@ -2,7 +2,8 @@
     ViewData["Title"] = "Home Page";
 }
 
-<div class="text-center">
-    <h1 class="display-4">Show page views and other information?</h1>
-    <p>If not that then maybe make event manager the home page?</p>
+<div class="card">
+    <div class="card-body">
+        <iframe src ="https://doorsopenpgh.org/"></iframe>
```

Comment:
This looks like it's going to confuse the hell out of someone if we let them see this. Can we make it basically say "Log In" or "Get Started" then it can take them to the buildings list?

Reply:
Yeah, I removed the frame and added a Get Started button